docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.9"
+services:
+  db:
+    image: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: app
+      POSTGRES_PASSWORD: app
```

Comment:
Just want to ensure ... This will only be used locally right? So we actually don't need to store it as ENV variables, right?

Reply:
Just used locally atm.

Reply:
Ok